app/Providers/RouteServiceProvider.php

```diff
@@ -33,72 +33,85 @@ class RouteServiceProvider extends ServiceProvider
 
     public function map()
     {
-        Route::middleware(Kernel::WEB)
-            ->get('/', function () {
-                return view('welcome');
-            })
+        Route::middleware(Kernel::ANY)
+            ->get(
+                '/',
+                function () {
+                    return view('welcome');
+                }
+            )
             ->name('login')
         ;
         $this->mapAuthRoutes();
         $this->mapAppRoutes();
         $this->mapApiRoutes();
 
+        Route::middleware(Kernel::ANY)
+            ->any('/test1', 'Adshares\Adserver\Http\Controllers\Controller@test')
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Adshares\Adserver\Http\Controllers\Controller@test" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)